doc/source/user_guide/new_meshing_workflows.rst

```diff
@@ -0,0 +1,462 @@
+.. _ref_user_guide_new_meshing_workflows:
+
+Use meshing workflows
+=====================
+PyFluent supports accessing all Fluent meshing functionalities, including
+guided meshing workflows.
+
+Watertight geometry meshing workflow
+------------------------------------
+This simple example shows how to use the watertight geometry meshing workflow.
+
+Import geometry
+~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    import ansys.fluent.core as pyfluent
+    from ansys.fluent.core import examples
+
+    import_file_name = examples.download_file('mixing_elbow.pmdb', 'pyfluent/mixing_elbow')
+    meshing = pyfluent.launch_fluent(
+        mode="meshing", precision='double', processor_count=2
+    )
+    watertight = meshing.watertight()
+    watertight.import_geometry.FileName.set_state(import_file_name)
+    watertight.import_geometry.LengthUnit.set_state('in')
+    watertight.import_geometry.Execute()
+
+Add local sizing
+~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.add_local_sizing.AddChildToTask()
+    watertight.add_local_sizing.Execute()
+
+Generate surface mesh
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.create_surface_mesh.CFDSurfaceMeshControls.MaxSize.set_state(0.3)
+    watertight.create_surface_mesh.Execute()
+
+Describe geometry
+~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.describe_geometry.UpdateChildTasks(SetupTypeChanged=False)
+    watertight.describe_geometry.SetupType.set_state("The geometry consists of only fluid regions with no voids")
+    watertight.describe_geometry.UpdateChildTasks(SetupTypeChanged=True)
+    watertight.describe_geometry.Execute()
+
+Update boundaries
+~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.update_boundaries.BoundaryLabelList.set_state(["wall-inlet"])
+    watertight.update_boundaries.BoundaryLabelTypeList.set_state(["wall"])
+    watertight.update_boundaries.OldBoundaryLabelList.set_state(["wall-inlet"])
+    watertight.update_boundaries.OldBoundaryLabelTypeList.set_state(["velocity-inlet"])
+     watertight.update_boundaries.Execute()
+
+Update regions
+~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.update_regions.Execute()
+
+Add boundary layers
+~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.add_boundary_layer.AddChildToTask()
+    watertight.add_boundary_layer.InsertCompoundChildTask()
+    meshing.workflow.TaskObject["smooth-transition_1"].Arguments = {
+        "BLControlName": "smooth-transition_1",
+    }
+    watertight.add_boundary_layer.Arguments = {}
```

Comment:
`Arguments` should not be exposed in this mode. it should be `arguments` for getting and for setting.